src/Microsoft.AspNetCore.Server.Kestrel.Https/Internal/ClosedStream.cs

```diff
@@ -5,12 +5,13 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Internal;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Https.Internal
 {
     internal class ClosedStream : Stream
     {
+        private static readonly Task<int> CompletedIntTask = Task.FromResult(default(int));
```

Comment:
That's pretty weird isn't it? Its explicitly a zero return - rather than a `default(int)`; also `CompletedIntTask` doesn't really specify what it is.